### DISCUSSION TITLE
/temps

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#1359] [Nouvelles idées]
Bonjour, Je voudrais vous soumettre une proposition qui est: L'ajout d'une commande /temps. Grâce à cette commande on pourrait voir en une seul fois le temps qu'il nous reste pour un prochain report le temps de trajet qui reste/ le temps qui reste pour le /daily et /guildedaily / le temps qui reste si nous avons fait en petfree... A très vite🥸

**Comment by SomniumWed** [comment]
> Bonjour, Je voudrais vous soumettre une proposition qui est: L'ajout d'une commande /temps. Grâce à cette commande on pourrait voir en une seul fois le temps qu'il nous reste pour un prochain report le temps de trajet qui reste/ le temps qui reste pour le /daily et /guildedaily / le temps qui reste si nous avons fait en petfree... A très vite🥸

Hello ! C'est déjà possible de voir le temps qu'il nous reste grâce à la même commande. Par contre je ne sais pas si on peut voir le temps pour le /petfree...

**Reply by Voisin1234** [reply]
Du coup pour répondre à ta question non nous ne pouvons pas voir le temps qu'il reste pour nourrir un animal. Et du coup ma proposition serait quand une commande on puisse voir tout le temps qu'il nous reste pour chaque interaction.
